src/field-renderer/messages.jsx

```diff
@@ -0,0 +1,10 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  'tooltip.text': {
```

Comment:
It's better to use CamelCase notation for message key
```suggestion
  tooltipText: { ... }
```

Reply:
We are using dot notation throughout the entire project